app/models/role.rb

```diff
@@ -864,155 +870,129 @@ def self.health_permissions_with_descriptions
         administrative: false,
-        categories: [
-        ],
+        categories: [],
       },
       can_approve_release: {
         description: 'Ability to approve release forms',
         administrative: false,
-        categories: [
-        ],
+        categories: [],
       },
       can_approve_participation: {
         description: 'Ability to approve participation forms',
         administrative: false,
-        categories: [
-        ],
+        categories: [],
       },
       can_edit_all_patient_items: {
         description: 'Unused',
         administrative: true,
-        categories: [
-        ],
+        categories: [],
       },
       can_edit_patient_items_for_own_agency: {
         description: 'Edit ability for patient records',
         administrative: false,
-        categories: [
-        ],
+        categories: [],
       },
       can_create_care_plans_for_own_agency: {
         description: 'Unused',
         administrative: false,
-        categories: [
-        ],
+        categories: [],
       },
       can_view_all_patients: {
         description: 'Unused',
         administrative: true,
-        categories: [
-        ],
+        categories: [],
       },
       can_view_patients_for_own_agency: {
         description: 'Allows access to patient records',
         administrative: false,
-        categories: [
-        ],
+        categories: [],
       }, # Read-only - not implemented as such yet
       can_add_case_management_notes: {
         description: 'Unused',
         administrative: false,
-        categories: [
-        ],
+        categories: [],
       },
       can_manage_accountable_care_organizations: {
         description: 'Administer ACO records',
         administrative: true,
-        categories: [
-        ],
+        categories: [],
       },
       can_view_member_health_reports: {
         description: 'Use for downloading individual member reports',
         administrative: true,
-        categories: [
-        ],
+        categories: [],
       },
       can_unsubmit_submitted_claims: {
         description: 'Can this user blank out the submitted date on QA, allowing resubmission?',
         administrative: true,
-        categories: [
-        ],
+        categories: [],
       },
       can_edit_health_emergency_contact_tracing: {
         description: 'Grants access to the contact tracing section when there is an active health emergency',
         administrative: false,
-        categories: [
-        ],
+        categories: [],
       },
     }
   end
 
   def self.ensure_permissions_exist
     Role.permissions.each do |permission|
-      unless ActiveRecord::Base.connection.column_exists?(:roles, permission)
-        ActiveRecord::Migration.add_column :roles, permission, :boolean, default: false
-      end
+      ActiveRecord::Migration.add_column :roles, permission, :boolean, default: false unless ActiveRecord::Base.connection.column_exists?(:roles, permission)
```

Comment:
Can you adjust to:
```
ActiveRecord::Migration.add_column(:roles, permission, :boolean, default: false) unless ActiveRecord::Base.connection.column_exists?```

Lack of parens makes this really unclear what's going on

Reply:
✅